Monika After Story/game/script-ch30.rpy

```diff
@@ -1109,6 +1109,7 @@ label ch30_autoload:
 
     # set this to None for now
     $ selected_greeting = None
+    $ mas_forced_exp = None
```

Comment:
can you make this a mailbox entry instead of var? means we don't have to set None after.

Reply:
Sure.